Dockerfile

```
@@ -0,0 +1,69 @@
+
+WORKDIR /app
+
+COPY --from=builder /app/target/release/nexum /usr/local/bin/nexum
+COPY nexum_ai ./nexum_ai
+
+RUN useradd --system --create-home --home-dir /app --shell /bin/bash nexumuser && \
+	chown -R nexumuser:nexumuser /app
+
+USER nexumuser
+
+RUN python3 -m venv .venv && \
```

Hadolint (2.14.0)

[warning] Pin versions in pip. Instead of `pip install <package>` use `pip install <package>==<version>` or `pip install --requirement <requirements file>`

(DL3013)